docs/integrations/log_auditbeat_linux.md

```diff
@@ -0,0 +1,335 @@
+  recursive: false
+
+- module: system
+  datasets:
+    - package   # Installed, updated, and removed packages
+  period: 2m
+
+- module: system
+  datasets:
+    - host      # General host information, e.g. uptime, IPs
+    - login     # User logins, logouts, and system boots.
+    - process   # Started and stopped processes
+    - socket    # Opened and closed sockets
+    - user      # User information
+
+
+  user.detect_password_changes: true
+
+# ================================== Outputs ===================================
+# ---------------------------- Elasticsearch Output ----------------------------
+output.elasticsearch:
+  enabled: false
+
+# -------------------------------- File Output ---------------------------------
+output.file:
+  enabled: true
+
+  # Configure JSON encoding
+  codec.json:
+    #pretty: false
+    # Configure escaping HTML symbols in strings.
+    #escape_html: false
+
+  # Path to the directory where to save the generated files.
+  path: "/tmp/auditbeat"
+  filename: auditbeat
+  rotate_every_kb: 10000
+  number_of_files: 7
+  permissions: 0600
+
+# =================================== Paths ====================================
+
+path.home: "/usr/share/auditbeat/bin"
+path.config: "/etc/auditbeat"
+path.data: "/var/lib/auditbeat"
+path.logs: "/var/log/auditbeat"
+
+# ================================== Template ==================================
+
+# Elasticsearch template settings
+setup.template.settings:
+  # A dictionary of settings to place into the settings.index dictionary
+  index:
+    number_of_shards: 1
+    #codec: best_compression
+
+# ================================== Logging ===================================
+
+logging.level: info
+
+logging.to_files: true
+logging.files:
+  path: /var/log/auditbeat
+  name: auditbeat
+  rotateeverybytes: 10485760 # = 10MB
+  keepfiles: 7
+  permissions: 0600
+
+```
+
+In order to make it easier, you can download the auditbeat configuration file:
+
+```bash
+$ wget -O /etc/auditbeat/auditbeat.yml https://docs.sekoia.io/integrations/configurations/auditbeat.yml
+```
+
+> Don't forget to set the right level of permission of the new `auditbeat.yml` if you edited a new one.
+> `sudo chmod 0600 auditbeat.yml`
+> Plus in this case ensure the owner of the file is root: `sudo chown root:root auditbeat.yml`
+
+Auditbeat comes with predefined assets for parsing, indexing, and visualizing your data. To load these assets:
+```bash
+sudo auditbeat -e
+```
+
+> If no error occurs, use `Ctrl + C` to stop running it in the terminal.
+
+Start AuditBeat with the folling command:
+```bash
+sudo systemctl restart auditbeat.service
+```
+
+> Check your logs in `/tmp/auditbeat/auditbeat` in JSON format.
+
+#### Configure the client Rsyslog
+
+In order to allow the rsyslog to work properly, please ensure the following packages are installed:
+
+```bash
+sudo apt install rsyslog
+```
+
+Settup a light client rsyslog by editing the `/etc/rsyslog.conf` file.

```

Comment:
This setup might break existing configuration. IMHO, we’d better provide a new configuration file (`/etc/rsyslog.d/auditbeat.conf` for example) that would read Auditbeat files and push logs to the log collector.

A file like this should do the job:

```
module(load="imfile" PollingInterval="10")              #needs to be done just once
input(type="imfile"
      File="/tmp/auditbeat/auditbeat"
      Tag="linux_auditbeat"
      Severity="info"
      Facility="local7")

if ($syslogtag contains 'linux_auditbeat') then {
     action(
         type="omfwd"
         protocol="tcp"
         target="YOUR_RSYSLOG_DESTINATION_SERVER"
         port="514"
         TCP_Framing="octet-counted"
     )
}
```

Reply:
@otetard I don't know the objectif is to ease the deployment with as less file configuration as possible. Hence the all in one rsyslog.conf

Reply:
Having to write a new file doesn’t add any complexity IMHO (it’s just a new file to add in `/etc/rsyslog.d/`). The other solution will introduce major regression/changes in the default logging configuration.